apis/python/src/tiledbsoma/util_arrow.py

```diff
@@ -121,36 +121,20 @@ def get_arrow_schema_from_tiledb_uri(
     return pa.schema(arrow_schema_dict)
 
 
-def ascii_to_unicode_pyarrow_readback(record_batch: pa.RecordBatch) -> pa.RecordBatch:
+def ascii_to_unicode_pyarrow_readback(table: pa.Table) -> pa.Table:
     """
     Implements the 'decode on read' part of our ASCII/Unicode logic
     """
     # TODO: COMMENT/LINK HEAVILY
-    names = [ofield.name for ofield in record_batch.schema]
+    names = [ofield.name for ofield in table.schema]
     new_fields = []
     for name in names:
-        old_field = record_batch[name]
-        if isinstance(old_field, pa.LargeBinaryArray):
+        old_field = table[name]
+        if len(old_field) > 0 and isinstance(old_field[0], pa.LargeBinaryScalar):
```

Comment:
I think the preferred pyarrow approach is to use the pyarrow.types helper functions rather than `isinstance`.  in this case, I believe that would be `pyarrow.types.is_large_binary`

Reply:
@bkmartinjr will do, but recall this is going away on the very next PR in this stack (#359)